### PR TITLE
fix: e2e/explore.test.js succeeds in offline mode

### PR DESCRIPTION
**Pull request by SgtPooki**
- test(e2e-explore): add apollo block fixtures
- test(e2e-explore): remove children fixtures
- test(e2e-explore): add project apollo archive test
- test(e2e-explore): use fixtures for XKCD archives

fixes https://github.com/ipfs/ipfs-webui/issues/2108
fixes https://github.com/ipfs/ipfs-webui/issues/2106

should unblock kubo CI failure: https://app.circleci.com/pipelines/github/ipfs/kubo/9385/workflows/dfd6c3bb-f2d0-4cb1-9b06-f7a21d528d42/jobs/101478

### CODE REVIEW

File: test/e2e/explore.test.js [hunk 1]
```diff
@@ -146,78 +173,106 @@ test.describe('Explore screen', () => {
...
+
+      await (await page.waitForSelector('"QmawceGscqN4o8Y8Fv26UUmB454kn2bnkXV5tEQYc4jBd6"')).click()
+      await testExploredCid({
+        fillOutForm: false,
+        page,
+        cid: 'QmawceGscqN4o8Y8Fv26UUmB454kn2bnkXV5tEQYc4jBd6',
+        humanReadableCID: 'base58btc - cidv0 - dag-pb - sha2-256~256~BB413C3DE0BA745523A3D701D6CB4283BCA7E187EC21556F4456036F692A5075',
+        type: 'dag-pb'
+      })
+      // await traverseChildren({ page, type: 'dag-pb' })
```
Comment (SgtPooki) [inline test/e2e/explore.test.js:213]: will remove

File: test/e2e/fixtures/generateFixtures.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  local fixture_cid=$1
+  shift
+  local fixture_path="$1/$fixture_cid"
+  echo -e "\$fixture_path: $fixture_path \n"
+  ipfs block get $fixture_cid > $fixture_path
+}
+
+# Example call:
+# test/e2e/fixtures/generateFixtures.sh QmSnuWmxptJZdLJpKRarxBMS2Ju2oANVrgbr2xWbie9b2D explore/blocks
+# saves passed root_cid block and it's children to the passed save_path
```
Comment (SgtPooki) [inline test/e2e/fixtures/generateFixtures.sh:16]: will update

**Review by SgtPooki — COMMENTED** [review]
self review

File: .gitattributes [hunk 3]
```diff
@@ -1 +1,3 @@
 * text=auto
+test/e2e/fixtures/explore/blocks/README.md text
+test/e2e/fixtures/explore/blocks/Qm* binary
```
Comment (lidel) [inline .gitattributes:3]: - we expect CARs and CIDv1 blocks in the future, let's add them here
- `linguist-generated` will collapse fixture data, making PRs with them  easier to review

```suggestion
test/e2e/fixtures/**/Qm* binary linguist-generated
test/e2e/fixtures/**/b* binary linguist-generated
test/e2e/fixtures/**/*.raw binary linguist-generated
test/e2e/fixtures/**/*.car binary linguist-generated
```

**Review by lidel — APPROVED** [review]
Thank you @SgtPooki!
lgtm, small suggestion inline

**Pull request merged**